roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ playbook_dir }}/{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
```

Comment:
I agree, let's remove it here so we allow more flexibility to the user. It may or may not break running playbooks from within another directory, but if it does, perhaps we can make changes to the `ansible.cfg` in that directory to accommodate. In addition, maybe we can just define the default `fetch_directory` variables to include `playbook_dir` instead.